### ISSUE TITLE
Speedtest server (id) not in the log anymore

### ISSUE THREAD

**Issue by KossikCZ** [roest01/docker-speedtest-analyser#65]
Hello guys,

I noticed after all the recent changes in this speedtest docker image the log (script) does not show the server speedtest was ran against anymore.

I have the SPEEDTEST_PARAMS set to execute against my ISP's server (ID 12722) but for past week my values of download got lower than usual which is a reason I got suspicious. When I run the speedtest from Docker command line (via Synology's DSM UI) against the server I set, I only get the error. That makes me believe the speedtest is not being actually executed against the predefined server.

Is it possible to get the server detail back into the log to see which server was actually used after all? Or anyone knowing why the server gets excluded in a first place (while the official speedtest-cli tool works with the same server just fine)?

![image](https://user-images.githubusercontent.com/48318841/104130508-7092a280-5371-11eb-8e01-ea38da69addd.png)

![image](https://user-images.githubusercontent.com/48318841/104130530-82744580-5371-11eb-9413-e5ca4da75803.png)

![image](https://user-images.githubusercontent.com/48318841/104130547-94ee7f00-5371-11eb-813c-2ba99bb7923d.png)

**Comment by GitBeto21** [comment]
Hi,
and I have the feeling that SPEEDTEST_PARAMS is not read or used. Because, no matter what I enter there, like for example "--no-download" nevertheless always the same comes out, in this case download is also tested.

**Comment by KossikCZ** [comment]
Yeah, that would actually make sense. I didn't test with other parameters but this would explain my observation too. 
It's a pity, the speedtest was working nicely until latest updates. :/
